colour/models/rgb/transfer_functions/canon_log.py

```diff
@@ -21,11 +25,15 @@
 .. [1]  Thorpe, L. (2012). CANON-LOG TRANSFER CHARACTERISTIC. Retrieved
         from http://downloads.canon.com/CDLC/\
 Canon-Log_Transfer_Characteristic_6-20-2012.pdf
+.. [2]  Canon. (n.d.). EOS C300 Mark II. Retrieved August 23, 2016, from
```

Comment:
I have updated the documentation to provide more details about the origin of the file. It is actually a download (I made a copy referenced in the issue).